benchmarks/nel/scripts/wiki/wiki_dump_api.py

```diff
@@ -24,17 +28,41 @@ def establish_db_connection() -> sqlite3.Connection:
     return db_conn
 
 
+def extract_demo_dump(filter_terms: Set[str]) -> None:
+    """Extracts small demo dump by parsing the Wiki dumps and keeping only those entities (and their articles)
+    containing any of the specified filter_terms. The retained entities and articles are written into
+    filter_terms (Set[str]): Terms having to appear in entity descriptions in order to be wrr
+    """
+
+    # entity_ids, entity_labels = wikidata.extract_demo_dump(
+    #     _paths["wikidata_dump"], _paths["filtered_wikidata_dump"], filter_terms
+    # )
+    # with open(_paths["filtered_entity_entity_info"], "wb") as file:
+    #     pickle.dump((entity_ids, entity_labels), file)
```

Comment:
If you're leaving this in for debugging or whatever that's fine, just going to flag it here in the review in case. I think there's a few more of these.

Reply:
Argh, commented that for testing. Will uncomment. Thanks!